lib/sass/calcite-web/components/_card.scss

```diff
@@ -43,6 +43,9 @@
 
 @mixin card-image-caption () {
   background: $opaque-white;
+  color: $off-black;
```

Comment:
Is this coming from previous `<figcaption>` styles in Calcite, or are these browser normalizations?